src/test/java/dataloader/JsonLoaderTest.java

```diff
@@ -0,0 +1,28 @@
+package dataloader;
+
+import org.junit.Test;
+import tech.tablesaw.api.Table;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+public class JsonLoaderTest {
+    /**
+     * Test Scenario: The application wants to read a json dataset
+     * but the String is invalid(empty String).
+     * Expected Output: JsonDataLoader throws an IOException.
+     */
+
+    @Test(expected = IOException.class)
+    public void testEmptyStringForHashMap() throws IOException {
+        JsonDataLoader loader = new JsonDataLoader();
+        loader.parseFile("nonExistent.json", new HashMap<String, Object>());
+    }
+
+    @Test(expected = IOException.class)
```

Comment:
Also write down the Test Scenario and the Expected output :)

Reply:
the description above I think covers both tests. that's why I didn't write it again. It's easy to do it in any case, if you think it's necessary.

Reply:
well although it would have been more uniform let's not be fanatics here...simple comment sth along the lines ```this is the same test  but for type...```

Reply:
did that for both tests.